src/services/audius-api-client/AudiusAPIClient.ts

```diff
@@ -431,7 +431,10 @@ class AudiusAPIClient {
     return adapted
   }
 
-  async getTrack({ id, currentUserId, unlistedArgs }: GetTrackArgs) {
+  async getTrack(
+    { id, currentUserId, unlistedArgs }: GetTrackArgs,
+    retry = true
```

Comment:
nit: any reason not to put retry in the GetTrackArgs?

Reply:
I figured it made more sense to leave out because it's not really part of the "args" themselves. maybe it could be part of a "config" object or something like that, but i don't think we'll use this all that much